### PR TITLE
Fix /server_info

### PR DESCRIPTION
**Pull request by rgs1**
When `/server_info` changed from a string to JSON (#4886), we forgot
to populate the restart epoch.

Signed-off-by: Raul Gutierrez Segales <rgs@pinterest.com>

### CODE REVIEW

**Comment by rgs1** [comment]
cc: @snowp @mattklein123

**Comment by mattklein123** [comment]
Great, thanks. Can we potentially add a test for this? I think it shouldn't be too hard to parse the JSON back into the proto and verify some of the outputs?

**Comment by snowp** [comment]
Thanks for fixing this. Iirc there is already a test, so we can probably add a check for this field there

**Comment by rgs1** [comment]
Yup, yup — adding a test.

**Comment by rgs1** [comment]
Kicked CI because of spurious mac build failure.
